utils/st2-check-register-pack-resources

```diff
@@ -35,7 +35,7 @@ PACK_PATH=$2
 
 source ${SCRIPT_PATH}/common.sh
 
-PACK_NAME=$(basename ${PACK_PATH})
+PACK_NAME=$(basename ${PACK_PATH} | cut -d "-" -f 1)
```

Comment:
Do you think it makes sense to change the name of this repo instead of hacking it like this? I think that we shouldn't do it in this PR, but perhaps file an issue on it and add a comment on it here, so that we remember it later?